### ISSUE TITLE
Single Automatic Payment 60 Days Prior to Departure

### ISSUE THREAD

**Issue by chris206** [206Tours/DB#2055]
On previous occasions, we were advised that it wasnt possible to do a single payment, 60 days prior to departure as is our manual company practice. After reviewing Authorize.net, I don't believe we have that limitation anymore.

In watching their ARB interface work, we could start and end a recurring payment at the 60 days prior mark.

So - client signs up 60 days prior, we ask whether they'd like to process the remaining balance automatically at 60 days prior to departure. They could select "YES", and if so maybe type their first and last name.

We would create a monthly subscription with the FULL remaining balance, start at the 60 days prior and end it 60 days prior. one payment would be made, 60 days prior. Theoretically, this would allow us to simplify the math and process considerable and meet our own standard. 

We could also simplify the price change review by automatically editing the automatic payment amount 90 days prior to departure and updating the amount remaining to be paid.

Can you confirm? Am I missing something?

**Comment by bradatkins** [comment]
Yes, this is possible. Technically it was possible with Elavon too.

**Comment by chris206** [comment]
Can we implement this right away while we work on the more complicated installment issue?

ALSO - if we are unable to get the DB to update the amount to charge (which I hope isnt true) - could we at least compare remaining balance in the DB to the amount set to be charged? if there is a discrepancy, a report would show accounting which orders need to be manually updated

**Comment by chris206** [comment]
how is this one going? Are we able to implement the single charge, on the date that final payment is due?

**Comment by bradatkins** [comment]
This isn't implemented, as it's a new enhancement and wasn't on the approved list of enhancements.
